modules/workflow-service-api/src/main/java/org/opencastproject/workflow/api/WorkflowIndexData.java

```diff
@@ -62,8 +63,12 @@ public class WorkflowIndexData {
   @Id
   private Long id;
   private int state;
+
+  @Column(name = "mediapackage_id", length = 128) //NB: This column definition needs to match WorkflowInstance!
   private String mediaPackageId;
-  private String organizationId;
+
+  @Column(name = "organization_id") //NB: This column definition needs to match WorkflowInstance!
+  private String organizationIds;
```

Comment:
Why the rename of `organizationId` to `organizationIds`?

Reply:
Hah, because I was trying to figure out which variable(s) generated the offending columns and forgot to change it back in the final commit 😅